storage/src/main/java/io/delta/storage/internal/LockUtils.java

```diff
@@ -0,0 +1,48 @@
+package io.delta.storage.internal;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.apache.hadoop.fs.Path;
+
+public final class LockUtils {
```

Comment:
I think its cleaner to make this a class with these two class methods.. rather than passing the hashmap to static methods. that bad encapsulation.

unless the pathlock objects are independently required by the logstore classes.